### DISCUSSION TITLE
Welcome to Frog Discussions!

### DISCUSSION THREAD

**Discussion by tmm** [wevm/frog-archived#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by berkayermis** [comment]
Could you please update your contributing docs. There is a [404](https://github.com/wevm/frog/blob/main/.github/CONTRIBUTING.md) page rn.

**Comment by airdropsinaga** [comment]
Good Project

**Comment by airdropsinaga** [comment]
> 
> ## 👋 Welcome!
>   We’re using Discussions as a place to connect with other members of our community. We hope that you:
>   * Ask questions you’re wondering about.
>   * Share ideas.
>   * Engage with other community members.
>   * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
>   To get started, comment below with an introduction of yourself and tell us about what you do with this community.
> 
> 
>
